src/styles/sidebar/components/sidebar-panel.scss

```diff
@@ -0,0 +1,50 @@
+.sidebar-panel {
+  border: solid 1px $grey-3;
+  border-radius: 2px;
+  font-family: $sans-font-family;
```

Comment:
Since I don't have a visual with this diff, I'm gonna assume the css is reasonable, but I do wonder, is this not inherited from body or somewhere? Seems to me font should be set near the root somewhere?

Reply:
I shall revisit this when I get to the place where I can see this visually!